tests/xfa/Makefile

```
@@ -2,8 +2,9 @@ include ../Makefile.tests_common
 
 include $(RIOTBASE)/Makefile.include
 
-all: static-test
-static-test: $(ELFFILE)
+BUILDDEPS += static-test
+
+static-test:
```

```suggestion
ifneq ($(BUILD_IN_DOCKER),1)
all: static-test
endif
static-test: $(ELFFILE)
```

This seems to work as well for me, IMO the rule does not belong in `BUILDDEPS`

---

> IMO the rule does not belong in BUILDDEPS

So why `all` depends on it ?
With your proposal, static-test is skipped by a regular build, no nothing is tested.

---

ups thats dumb